designs/aggregated-cluster-view/design.md

```diff
@@ -0,0 +1,353 @@
+Aggregated Cluster View Design
+==============================
+
+## Introduction
+Currently Helix organize information by cluster - clusters are autonomous entities that holds resource / node information.
+In real practice, a helix client might need to access aggregated information of helix clusters from different data center regions for management or coordination purpose.
+This design proposes a service in Helix ecosystem for clients to retrieve cross-datacenter information in a more efficient way. 
+
+
+## Problem Statement
+We identified a couple of use cases for accessing cross datacenter information. [Ambry](https://github.com/linkedin/ambry) is one of them.
```

Comment:
can you expand more on why Ambry needs this feature?

Reply:
Sure (will also update design doc about it).

Ambry uses Helix spectator in both their router (for retrying get requests remotely if failed locally) and storage node (for data replication purpose). Given the amount of clients that need global information, it would be more cost-effective for them if aggregated information are provided locally.